api/incubator/src/test/java/io/opentelemetry/api/incubator/trace/ExtendedDefaultTracerTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.api.incubator.trace;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.AbstractDefaultTracerTest;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.api.trace.TracerProvider;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+public class ExtendedDefaultTracerTest extends AbstractDefaultTracerTest {
```

Comment:
#nit: remove public modifiers on all these test classes and methods annotated with `@Test`
```suggestion
class ExtendedDefaultTracerTest extends AbstractDefaultTracerTest {
```